lib/hcc/CMakeLists.txt

```
@@ -1,26 +1,24 @@
-project( libkotekan_hcc )
+project(libkotekan_hcc)
```

```suggestion
project(kotekan_hcc)
```
Consistency with other libraries.